kitsune/users/jinja2/users/includes/notification_macros.html

```diff
@@ -0,0 +1,51 @@
+{% macro notification_success(static_url) -%}
+<div class="notification success">
+  <img class="ff-logo" src="{{ static_url }}sumo/img/firefox-logo.svg" alt="{{ _('Firefox logo') }}"/>
+  <div class="notification-content">
+    <h3 class="notification-title">{{ _('Good news!') }}</h3>
+    <p>{{ _('You can now log in to Mozilla Support using Firefox Accounts.') }}</p>
+  </div>
+  <a href="javascript:;" class="close-button"><span class="show-for-sr-only">{{ _('Close') }}</span></a>
+</div>
+{%- endmacro %}
+
+{% macro notification_welcome(static_url) -%}
+<div class="notification alert">
+  <img class="ff-logo" src="{{ static_url }}sumo/img/firefox-logo.svg" alt="{{ _('Firefox logo') }}"/>
+  <div class="notification-content">
+    <h3 class="notification-title">{{ _('Welcome!') }}</h3>
+    <p>{{ _('You are now logged in using Firefox Accounts. Already have a different Mozilla Support Account?') }}</p>
+  </div>
+  <div class="notification-actions">
+    <a href="https://support.mozilla.org/en-US/kb/firefox-accounts-mozilla-support-faq" class="button dark">{{ _('Read More') }}</a>
+  </div>
+  <a href="javascript:;" class="close-button"><span class="show-for-sr-only">{{ _('Close') }}</span></a>
+</div>
+{%- endmacro %}
+
+{% macro notification_warning(static_url) -%}
+<div class="notification alert">
+  <img class="ff-logo" src="{{ static_url }}sumo/img/firefox-logo.svg" alt="{{ _('Firefox logo') }}"/>
+  <div class="notification-content">
+    <h3 class="notification-title">{{ _('Heads up!') }}</h3>
+    <p>{{ _('Soon, Firefox Accounts will replace Mozilla Support logins. You can upgrade your existing Mozilla account to Firefox Accounts today.') }} <a href="https://support.mozilla.org/en-US/kb/firefox-accounts-mozilla-support-faq">{{ _('Read more') }}</a>.</p>
+  </div>
+  <div class="notification-actions">
+    <a href="{{ url('users.fxa_authentication_init') }}" class="button">{{ _('Continue with Firefox Accounts') }}</a>
+  </div>
+</div>
+{%- endmacro %}
+
+{% macro notification_already_migrated(static_url) -%}
+<div id="fxa-login-error" class="notification alert">
+  <img class="ff-logo" src="{{ static_url }}sumo/img/firefox-logo.svg" alt="{{ _('Firefox logo') }}"/>
+  <div class="notification-content">
+    <h3 class="notification-title">{{ _('Sorry!') }}</h3>
+    <p>{{ _('You already have a Firefox Accounts. Please use your Firefox Account instead of your Mozilla Support
+    account.') }}</p>
+  </div>
+  <div class="notification-actions">
+    <a href="{{ url('users.fxa_authentication_init') }}" class="button">{{ _('Continue with Firefox Accounts') }}</a>
+  </div>
```

Comment:
Probably we don't need this link here since it's right above the one that asks the user to login. Thoughts?

Reply:
Agreed. I'll remove